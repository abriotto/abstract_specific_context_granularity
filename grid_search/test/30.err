  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  5%|#########4                                                                                                                                                                                  | 5/99 [00:00<00:02, 45.53it/s] 11%|####################7                                                                                                                                                                      | 11/99 [00:00<00:01, 50.93it/s] 20%|#####################################7                                                                                                                                                     | 20/99 [00:00<00:01, 65.02it/s] 29%|######################################################7                                                                                                                                    | 29/99 [00:00<00:00, 71.13it/s] 37%|#####################################################################8                                                                                                                     | 37/99 [00:00<00:00, 71.78it/s] 46%|######################################################################################8                                                                                                    | 46/99 [00:00<00:00, 73.54it/s] 55%|#####################################################################################################9                                                                                     | 54/99 [00:00<00:00, 69.74it/s] 62%|###################################################################################################################2                                                                       | 61/99 [00:00<00:00, 67.85it/s] 69%|################################################################################################################################4                                                          | 68/99 [00:01<00:00, 61.18it/s] 76%|#############################################################################################################################################6                                             | 75/99 [00:01<00:00, 61.55it/s] 84%|############################################################################################################################################################7                              | 83/99 [00:01<00:00, 64.94it/s] 91%|##########################################################################################################################################################################                 | 90/99 [00:01<00:00, 62.02it/s] 98%|#######################################################################################################################################################################################2   | 97/99 [00:01<00:00, 59.91it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 63.13it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 24%|#############################################1                                                                                                                                              | 6/25 [00:00<00:00, 54.49it/s] 52%|#################################################################################################2                                                                                         | 13/25 [00:00<00:00, 55.25it/s] 92%|############################################################################################################################################################################               | 23/25 [00:00<00:00, 70.64it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 63.57it/s]
