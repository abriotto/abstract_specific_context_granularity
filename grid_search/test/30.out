# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=64", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.97", "--temperature=1"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7021129131317139, "acc": 0.5048945546150208, "length": 3.6130518913269043, "mode": "train", "epoch": 1}
{"loss": 0.6905891299247742, "acc": 0.5228938460350037, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6970083117485046, "acc": 0.514394998550415, "length": 3.365705966949463, "mode": "train", "epoch": 2}
{"loss": 0.6930336952209473, "acc": 0.5228937864303589, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6925963759422302, "acc": 0.520297646522522, "length": 2.7627851963043213, "mode": "train", "epoch": 3}
{"loss": 0.6974897384643555, "acc": 0.47985345125198364, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6905456185340881, "acc": 0.5191009640693665, "length": 2.0849716663360596, "mode": "train", "epoch": 4}
{"loss": 0.6970192790031433, "acc": 0.4880952835083008, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6803855299949646, "acc": 0.5744518637657166, "length": 2.4574904441833496, "mode": "train", "epoch": 5}
{"loss": 0.6617382764816284, "acc": 0.6411730051040649, "length": 2.087912082672119, "mode": "test", "epoch": 5}
{"loss": 0.6528162956237793, "acc": 0.6464359164237976, "length": 2.5215728282928467, "mode": "train", "epoch": 6}
{"loss": 0.6337295770645142, "acc": 0.653606116771698, "length": 2.2527472972869873, "mode": "test", "epoch": 6}
{"loss": 0.6112025380134583, "acc": 0.6895996332168579, "length": 2.597196578979492, "mode": "train", "epoch": 7}
{"loss": 0.5870721936225891, "acc": 0.6972441673278809, "length": 2.703296661376953, "mode": "test", "epoch": 7}
{"loss": 0.5748431086540222, "acc": 0.7158007025718689, "length": 2.8305230140686035, "mode": "train", "epoch": 8}
{"loss": 0.5572421550750732, "acc": 0.7352028489112854, "length": 3.0164835453033447, "mode": "test", "epoch": 8}
{"loss": 0.5398463606834412, "acc": 0.75270676612854, "length": 3.1383895874023438, "mode": "train", "epoch": 9}
{"loss": 0.5238432884216309, "acc": 0.757687509059906, "length": 3.219780206680298, "mode": "test", "epoch": 9}
{"loss": 0.5070838928222656, "acc": 0.7773481607437134, "length": 3.3919363021850586, "mode": "train", "epoch": 10}
{"loss": 0.4817110300064087, "acc": 0.7907981276512146, "length": 3.538461446762085, "mode": "test", "epoch": 10}
{"loss": 0.4791090190410614, "acc": 0.795101523399353, "length": 3.480255126953125, "mode": "train", "epoch": 11}
{"loss": 0.4562228322029114, "acc": 0.8010192513465881, "length": 3.5054945945739746, "mode": "test", "epoch": 11}
{"loss": 0.4446142017841339, "acc": 0.8151306509971619, "length": 3.4869699478149414, "mode": "train", "epoch": 12}
{"loss": 0.44676119089126587, "acc": 0.8085582852363586, "length": 3.5439560413360596, "mode": "test", "epoch": 12}
{"loss": 0.4153750538825989, "acc": 0.8327374458312988, "length": 3.511194944381714, "mode": "train", "epoch": 13}
{"loss": 0.42484205961227417, "acc": 0.8222336769104004, "length": 3.4945054054260254, "mode": "test", "epoch": 13}
{"loss": 0.39956703782081604, "acc": 0.8335021734237671, "length": 3.5809717178344727, "mode": "train", "epoch": 14}
{"loss": 0.4088374972343445, "acc": 0.8271424770355225, "length": 3.461538553237915, "mode": "test", "epoch": 14}
{"loss": 0.38395872712135315, "acc": 0.8447213172912598, "length": 3.521583318710327, "mode": "train", "epoch": 15}
{"loss": 0.4122023582458496, "acc": 0.8276039958000183, "length": 3.472527503967285, "mode": "test", "epoch": 15}
{"loss": 0.3698749542236328, "acc": 0.8425265550613403, "length": 3.530975103378296, "mode": "train", "epoch": 16}
{"loss": 0.41030043363571167, "acc": 0.8210835456848145, "length": 3.6593406200408936, "mode": "test", "epoch": 16}
{"loss": 0.3592936396598816, "acc": 0.8463725447654724, "length": 3.6283578872680664, "mode": "train", "epoch": 17}
{"loss": 0.40522855520248413, "acc": 0.8187772035598755, "length": 3.604395627975464, "mode": "test", "epoch": 17}
{"loss": 0.34343305230140686, "acc": 0.857724666595459, "length": 3.6133899688720703, "mode": "train", "epoch": 18}
{"loss": 0.38776475191116333, "acc": 0.8245536088943481, "length": 3.6318681240081787, "mode": "test", "epoch": 18}
{"loss": 0.3369129002094269, "acc": 0.8572680950164795, "length": 3.6137332916259766, "mode": "train", "epoch": 19}
{"loss": 0.39428871870040894, "acc": 0.8253825306892395, "length": 3.6373627185821533, "mode": "test", "epoch": 19}
{"loss": 0.3257588744163513, "acc": 0.8673456907272339, "length": 3.650602102279663, "mode": "train", "epoch": 20}
{"loss": 0.39721155166625977, "acc": 0.8272373676300049, "length": 3.615384578704834, "mode": "test", "epoch": 20}
{"loss": 0.3309267461299896, "acc": 0.8646813035011292, "length": 3.6487350463867188, "mode": "train", "epoch": 21}
{"loss": 0.38211914896965027, "acc": 0.8351436853408813, "length": 3.6703295707702637, "mode": "test", "epoch": 21}
{"loss": 0.31801024079322815, "acc": 0.8694412708282471, "length": 3.626831531524658, "mode": "train", "epoch": 22}
{"loss": 0.4028712213039398, "acc": 0.8296354413032532, "length": 3.6098902225494385, "mode": "test", "epoch": 22}
{"loss": 0.31701046228408813, "acc": 0.8714238405227661, "length": 3.6637518405914307, "mode": "train", "epoch": 23}
{"loss": 0.38628190755844116, "acc": 0.8296210169792175, "length": 3.626373529434204, "mode": "test", "epoch": 23}
{"loss": 0.3179014325141907, "acc": 0.8697476387023926, "length": 3.6723928451538086, "mode": "train", "epoch": 24}
{"loss": 0.37036365270614624, "acc": 0.8472826480865479, "length": 3.604395627975464, "mode": "test", "epoch": 24}
{"loss": 0.30775654315948486, "acc": 0.8788679838180542, "length": 3.6202332973480225, "mode": "train", "epoch": 25}
{"loss": 0.3700296878814697, "acc": 0.8554311990737915, "length": 3.6098902225494385, "mode": "test", "epoch": 25}
{"loss": 0.3064500093460083, "acc": 0.8795033693313599, "length": 3.639373540878296, "mode": "train", "epoch": 26}
{"loss": 0.35016536712646484, "acc": 0.8559403419494629, "length": 3.6593406200408936, "mode": "test", "epoch": 26}
{"loss": 0.28580442070961, "acc": 0.8872560262680054, "length": 3.6907410621643066, "mode": "train", "epoch": 27}
{"loss": 0.3521190583705902, "acc": 0.855475127696991, "length": 3.6978023052215576, "mode": "test", "epoch": 27}
{"loss": 0.28963735699653625, "acc": 0.882099986076355, "length": 3.676753044128418, "mode": "train", "epoch": 28}
{"loss": 0.3464012145996094, "acc": 0.8566918969154358, "length": 3.6098902225494385, "mode": "test", "epoch": 28}
{"loss": 0.28592991828918457, "acc": 0.8883720636367798, "length": 3.637742757797241, "mode": "train", "epoch": 29}
{"loss": 0.3480316996574402, "acc": 0.8555403351783752, "length": 3.6098902225494385, "mode": "test", "epoch": 29}
{"loss": 0.2766979932785034, "acc": 0.8936663269996643, "length": 3.669288396835327, "mode": "train", "epoch": 30}
{"loss": 0.33646053075790405, "acc": 0.8587582111358643, "length": 3.664835214614868, "mode": "test", "epoch": 30}
{"loss": 0.26437053084373474, "acc": 0.8960152268409729, "length": 3.7013611793518066, "mode": "train", "epoch": 31}
{"loss": 0.32510730624198914, "acc": 0.8578832149505615, "length": 3.7087912559509277, "mode": "test", "epoch": 31}
{"loss": 0.26857954263687134, "acc": 0.8937506675720215, "length": 3.6956379413604736, "mode": "train", "epoch": 32}
{"loss": 0.34543269872665405, "acc": 0.8680518865585327, "length": 3.5989010334014893, "mode": "test", "epoch": 32}
{"loss": 0.2715654969215393, "acc": 0.8939437866210938, "length": 3.6751956939697266, "mode": "train", "epoch": 33}
{"loss": 0.32221972942352295, "acc": 0.8740584254264832, "length": 3.6593406200408936, "mode": "test", "epoch": 33}
{"loss": 0.2582029402256012, "acc": 0.8988347053527832, "length": 3.6739799976348877, "mode": "train", "epoch": 34}
{"loss": 0.33315446972846985, "acc": 0.8681737780570984, "length": 3.6703295707702637, "mode": "test", "epoch": 34}
{"loss": 0.2613818049430847, "acc": 0.8983416557312012, "length": 3.698709011077881, "mode": "train", "epoch": 35}
{"loss": 0.3476184010505676, "acc": 0.8619712591171265, "length": 3.6593406200408936, "mode": "test", "epoch": 35}
{"loss": 0.25624626874923706, "acc": 0.9033112525939941, "length": 3.6870055198669434, "mode": "train", "epoch": 36}
{"loss": 0.32944363355636597, "acc": 0.8709014058113098, "length": 3.7087912559509277, "mode": "test", "epoch": 36}
{"loss": 0.2546977698802948, "acc": 0.9040394425392151, "length": 3.7279605865478516, "mode": "train", "epoch": 37}
{"loss": 0.33381637930870056, "acc": 0.8708038330078125, "length": 3.6373627185821533, "mode": "test", "epoch": 37}
{"loss": 0.24935606122016907, "acc": 0.8994199633598328, "length": 3.6866257190704346, "mode": "train", "epoch": 38}
{"loss": 0.3341887593269348, "acc": 0.8679221272468567, "length": 3.6593406200408936, "mode": "test", "epoch": 38}
{"loss": 0.24755263328552246, "acc": 0.9040694832801819, "length": 3.6827287673950195, "mode": "train", "epoch": 39}
{"loss": 0.31227362155914307, "acc": 0.8770589232444763, "length": 3.6593406200408936, "mode": "test", "epoch": 39}
{"loss": 0.23612266778945923, "acc": 0.9083829522132874, "length": 3.720857858657837, "mode": "train", "epoch": 40}
{"loss": 0.32859742641448975, "acc": 0.8698245286941528, "length": 3.741758346557617, "mode": "test", "epoch": 40}
{"loss": 0.2392835170030594, "acc": 0.909078061580658, "length": 3.708966016769409, "mode": "train", "epoch": 41}
{"loss": 0.3254297375679016, "acc": 0.8706842064857483, "length": 3.6703295707702637, "mode": "test", "epoch": 41}
{"loss": 0.23215337097644806, "acc": 0.9120000004768372, "length": 3.7037785053253174, "mode": "train", "epoch": 42}
{"loss": 0.33865606784820557, "acc": 0.8710991144180298, "length": 3.642857074737549, "mode": "test", "epoch": 42}
{"loss": 0.23807017505168915, "acc": 0.9113717079162598, "length": 3.657691478729248, "mode": "train", "epoch": 43}
{"loss": 0.333622545003891, "acc": 0.8647869229316711, "length": 3.6703295707702637, "mode": "test", "epoch": 43}
{"loss": 0.24074570834636688, "acc": 0.9096794128417969, "length": 3.719203233718872, "mode": "train", "epoch": 44}
{"loss": 0.324751615524292, "acc": 0.8761383891105652, "length": 3.6978023052215576, "mode": "test", "epoch": 44}
{"loss": 0.22142797708511353, "acc": 0.9160990118980408, "length": 3.7232227325439453, "mode": "train", "epoch": 45}
{"loss": 0.3360278606414795, "acc": 0.8684360384941101, "length": 3.7472527027130127, "mode": "test", "epoch": 45}
{"loss": 0.2209482342004776, "acc": 0.9142757058143616, "length": 3.7443130016326904, "mode": "train", "epoch": 46}
{"loss": 0.3196955919265747, "acc": 0.8772861361503601, "length": 3.653846263885498, "mode": "test", "epoch": 46}
{"loss": 0.22566838562488556, "acc": 0.9120113849639893, "length": 3.7131881713867188, "mode": "train", "epoch": 47}
{"loss": 0.32918649911880493, "acc": 0.8783413171768188, "length": 3.681318759918213, "mode": "test", "epoch": 47}
{"loss": 0.21486027538776398, "acc": 0.9183659553527832, "length": 3.7164933681488037, "mode": "train", "epoch": 48}
{"loss": 0.33374810218811035, "acc": 0.8696215152740479, "length": 3.719780206680298, "mode": "test", "epoch": 48}
{"loss": 0.21978776156902313, "acc": 0.9156377911567688, "length": 3.756546974182129, "mode": "train", "epoch": 49}
{"loss": 0.3147388696670532, "acc": 0.8814627528190613, "length": 3.719780206680298, "mode": "test", "epoch": 49}
{"loss": 0.21581557393074036, "acc": 0.9166660904884338, "length": 3.7493953704833984, "mode": "train", "epoch": 50}
{"loss": 0.3119826912879944, "acc": 0.88393235206604, "length": 3.7087912559509277, "mode": "test", "epoch": 50}
{"loss": 0.21544957160949707, "acc": 0.9228043556213379, "length": 3.7301692962646484, "mode": "train", "epoch": 51}
{"loss": 0.3168070912361145, "acc": 0.8843923807144165, "length": 3.719780206680298, "mode": "test", "epoch": 51}
{"loss": 0.20600415766239166, "acc": 0.921972393989563, "length": 3.7661616802215576, "mode": "train", "epoch": 52}
{"loss": 0.31396907567977905, "acc": 0.8905338644981384, "length": 3.7142856121063232, "mode": "test", "epoch": 52}
{"loss": 0.22199507057666779, "acc": 0.9206234216690063, "length": 3.744008779525757, "mode": "train", "epoch": 53}
{"loss": 0.3224870562553406, "acc": 0.8857442140579224, "length": 3.7142856121063232, "mode": "test", "epoch": 53}
{"loss": 0.2107115536928177, "acc": 0.9193381071090698, "length": 3.7320637702941895, "mode": "train", "epoch": 54}
{"loss": 0.32330751419067383, "acc": 0.882830023765564, "length": 3.6868131160736084, "mode": "test", "epoch": 54}
{"loss": 0.20943601429462433, "acc": 0.9228163957595825, "length": 3.714879035949707, "mode": "train", "epoch": 55}
{"loss": 0.32973751425743103, "acc": 0.8825370073318481, "length": 3.719780206680298, "mode": "test", "epoch": 55}
{"loss": 0.20452116429805756, "acc": 0.9210833311080933, "length": 3.7832326889038086, "mode": "train", "epoch": 56}
{"loss": 0.32435160875320435, "acc": 0.8858628869056702, "length": 3.7637362480163574, "mode": "test", "epoch": 56}
{"loss": 0.2006511241197586, "acc": 0.9230608940124512, "length": 3.7585396766662598, "mode": "train", "epoch": 57}
{"loss": 0.3304385840892792, "acc": 0.8869327902793884, "length": 3.7637362480163574, "mode": "test", "epoch": 57}
{"loss": 0.19897998869419098, "acc": 0.9243733882904053, "length": 3.756514549255371, "mode": "train", "epoch": 58}
{"loss": 0.3280234634876251, "acc": 0.8873064517974854, "length": 3.7142856121063232, "mode": "test", "epoch": 58}
{"loss": 0.2050100713968277, "acc": 0.9171707630157471, "length": 3.7313644886016846, "mode": "train", "epoch": 59}
{"loss": 0.33873140811920166, "acc": 0.8839905858039856, "length": 3.7142856121063232, "mode": "test", "epoch": 59}
{"loss": 0.20464523136615753, "acc": 0.9200170040130615, "length": 3.7374866008758545, "mode": "train", "epoch": 60}
{"loss": 0.3270447254180908, "acc": 0.8853244781494141, "length": 3.7142856121063232, "mode": "test", "epoch": 60}
{"loss": 0.20372502505779266, "acc": 0.922478437423706, "length": 3.7328379154205322, "mode": "train", "epoch": 61}
{"loss": 0.3212229609489441, "acc": 0.8826058506965637, "length": 3.7087912559509277, "mode": "test", "epoch": 61}
{"loss": 0.19734624028205872, "acc": 0.927899956703186, "length": 3.7500224113464355, "mode": "train", "epoch": 62}
{"loss": 0.31218063831329346, "acc": 0.8885481953620911, "length": 3.758241653442383, "mode": "test", "epoch": 62}
{"loss": 0.19576236605644226, "acc": 0.926234245300293, "length": 3.7705917358398438, "mode": "train", "epoch": 63}
{"loss": 0.3398091197013855, "acc": 0.8816365003585815, "length": 3.769230842590332, "mode": "test", "epoch": 63}
{"loss": 0.18497343361377716, "acc": 0.9275323152542114, "length": 3.8071324825286865, "mode": "train", "epoch": 64}
{"loss": 0.3451460301876068, "acc": 0.8813371062278748, "length": 3.7912087440490723, "mode": "test", "epoch": 64}
{"loss": 0.2028626650571823, "acc": 0.9216316938400269, "length": 3.774613380432129, "mode": "train", "epoch": 65}
{"loss": 0.34733253717422485, "acc": 0.8883679509162903, "length": 3.796703338623047, "mode": "test", "epoch": 65}
{"loss": 0.20134186744689941, "acc": 0.923868715763092, "length": 3.774998426437378, "mode": "train", "epoch": 66}
{"loss": 0.3074418008327484, "acc": 0.8886345028877258, "length": 3.719780206680298, "mode": "test", "epoch": 66}
{"loss": 0.19358405470848083, "acc": 0.9269801378250122, "length": 3.752711057662964, "mode": "train", "epoch": 67}
{"loss": 0.32607802748680115, "acc": 0.8834500312805176, "length": 3.758241653442383, "mode": "test", "epoch": 67}
{"loss": 0.1913699358701706, "acc": 0.9276652336120605, "length": 3.7577993869781494, "mode": "train", "epoch": 68}
{"loss": 0.3129649758338928, "acc": 0.8911114931106567, "length": 3.7362637519836426, "mode": "test", "epoch": 68}
{"loss": 0.18654049932956696, "acc": 0.924614429473877, "length": 3.7768595218658447, "mode": "train", "epoch": 69}
{"loss": 0.31783533096313477, "acc": 0.8796991109848022, "length": 3.7527472972869873, "mode": "test", "epoch": 69}
{"loss": 0.18819430470466614, "acc": 0.9247775673866272, "length": 3.7610697746276855, "mode": "train", "epoch": 70}
{"loss": 0.3023867607116699, "acc": 0.8887568712234497, "length": 3.7637362480163574, "mode": "test", "epoch": 70}
{"loss": 0.18835468590259552, "acc": 0.9265494346618652, "length": 3.7712810039520264, "mode": "train", "epoch": 71}
{"loss": 0.305472731590271, "acc": 0.8842571377754211, "length": 3.741758346557617, "mode": "test", "epoch": 71}
{"loss": 0.1848534494638443, "acc": 0.928680419921875, "length": 3.7689874172210693, "mode": "train", "epoch": 72}
{"loss": 0.3386935889720917, "acc": 0.8779762387275696, "length": 3.758241653442383, "mode": "test", "epoch": 72}
{"loss": 0.1922069936990738, "acc": 0.9288344383239746, "length": 3.7703843116760254, "mode": "train", "epoch": 73}
{"loss": 0.3496870696544647, "acc": 0.8837140202522278, "length": 3.7142856121063232, "mode": "test", "epoch": 73}
{"loss": 0.19246095418930054, "acc": 0.9265893697738647, "length": 3.753812789916992, "mode": "train", "epoch": 74}
{"loss": 0.31000590324401855, "acc": 0.8930649161338806, "length": 3.7142856121063232, "mode": "test", "epoch": 74}
{"loss": 0.18923571705818176, "acc": 0.9306273460388184, "length": 3.7482662200927734, "mode": "train", "epoch": 75}
{"loss": 0.3281784653663635, "acc": 0.8921968340873718, "length": 3.730769157409668, "mode": "test", "epoch": 75}
{"loss": 0.1837337613105774, "acc": 0.9275269508361816, "length": 3.7668874263763428, "mode": "train", "epoch": 76}
{"loss": 0.3134334087371826, "acc": 0.8875565528869629, "length": 3.780219793319702, "mode": "test", "epoch": 76}
{"loss": 0.1812792867422104, "acc": 0.9310437440872192, "length": 3.7784221172332764, "mode": "train", "epoch": 77}
{"loss": 0.30738553404808044, "acc": 0.8871347904205322, "length": 3.7747251987457275, "mode": "test", "epoch": 77}
{"loss": 0.18588872253894806, "acc": 0.9299875497817993, "length": 3.774538278579712, "mode": "train", "epoch": 78}
{"loss": 0.3242253363132477, "acc": 0.8868973255157471, "length": 3.7362637519836426, "mode": "test", "epoch": 78}
{"loss": 0.17747586965560913, "acc": 0.9333318471908569, "length": 3.7454004287719727, "mode": "train", "epoch": 79}
{"loss": 0.3078005909919739, "acc": 0.8884130716323853, "length": 3.7087912559509277, "mode": "test", "epoch": 79}
{"loss": 0.17684300243854523, "acc": 0.9354299306869507, "length": 3.7495594024658203, "mode": "train", "epoch": 80}
{"loss": 0.30917391180992126, "acc": 0.8933225274085999, "length": 3.730769157409668, "mode": "test", "epoch": 80}
{"loss": 0.17758414149284363, "acc": 0.9344236850738525, "length": 3.7875728607177734, "mode": "train", "epoch": 81}
{"loss": 0.3236605226993561, "acc": 0.8826603293418884, "length": 3.780219793319702, "mode": "test", "epoch": 81}
{"loss": 0.17491085827350616, "acc": 0.9347900748252869, "length": 3.7861475944519043, "mode": "train", "epoch": 82}
{"loss": 0.31837335228919983, "acc": 0.8889646530151367, "length": 3.758241653442383, "mode": "test", "epoch": 82}
{"loss": 0.16783443093299866, "acc": 0.9415093660354614, "length": 3.773437023162842, "mode": "train", "epoch": 83}
{"loss": 0.3182339668273926, "acc": 0.8938942551612854, "length": 3.7637362480163574, "mode": "test", "epoch": 83}
{"loss": 0.16702468693256378, "acc": 0.937929630279541, "length": 3.7885639667510986, "mode": "train", "epoch": 84}
{"loss": 0.3344404399394989, "acc": 0.8892513513565063, "length": 3.741758346557617, "mode": "test", "epoch": 84}
{"loss": 0.17504313588142395, "acc": 0.9357457160949707, "length": 3.7700977325439453, "mode": "train", "epoch": 85}
{"loss": 0.32179975509643555, "acc": 0.8988964557647705, "length": 3.730769157409668, "mode": "test", "epoch": 85}
{"loss": 0.18357060849666595, "acc": 0.9284341931343079, "length": 3.7719407081604004, "mode": "train", "epoch": 86}
{"loss": 0.3051100969314575, "acc": 0.8947363495826721, "length": 3.758241653442383, "mode": "test", "epoch": 86}
{"loss": 0.17636515200138092, "acc": 0.9310426712036133, "length": 3.777998447418213, "mode": "train", "epoch": 87}
{"loss": 0.2834585905075073, "acc": 0.9062431454658508, "length": 3.780219793319702, "mode": "test", "epoch": 87}
{"loss": 0.16569216549396515, "acc": 0.9391134977340698, "length": 3.8071746826171875, "mode": "train", "epoch": 88}
{"loss": 0.3132340908050537, "acc": 0.894018292427063, "length": 3.8406593799591064, "mode": "test", "epoch": 88}
{"loss": 0.1728314757347107, "acc": 0.9339082837104797, "length": 3.8321359157562256, "mode": "train", "epoch": 89}
{"loss": 0.32194095849990845, "acc": 0.8920161724090576, "length": 3.780219793319702, "mode": "test", "epoch": 89}
{"loss": 0.16039776802062988, "acc": 0.9415808320045471, "length": 3.801035165786743, "mode": "train", "epoch": 90}
{"loss": 0.3124787509441376, "acc": 0.9025934934616089, "length": 3.758241653442383, "mode": "test", "epoch": 90}
{"loss": 0.1580691784620285, "acc": 0.9426339268684387, "length": 3.777557849884033, "mode": "train", "epoch": 91}
{"loss": 0.2826572060585022, "acc": 0.9089288115501404, "length": 3.7637362480163574, "mode": "test", "epoch": 91}
{"loss": 0.1555396318435669, "acc": 0.9446368217468262, "length": 3.7893927097320557, "mode": "train", "epoch": 92}
{"loss": 0.2866451144218445, "acc": 0.9096174240112305, "length": 3.7912087440490723, "mode": "test", "epoch": 92}
{"loss": 0.15653914213180542, "acc": 0.9379708766937256, "length": 3.7925939559936523, "mode": "train", "epoch": 93}
{"loss": 0.2983390986919403, "acc": 0.8990443348884583, "length": 3.7857143878936768, "mode": "test", "epoch": 93}
{"loss": 0.15069900453090668, "acc": 0.9449312686920166, "length": 3.7857422828674316, "mode": "train", "epoch": 94}
{"loss": 0.3044356107711792, "acc": 0.8938561677932739, "length": 3.7637362480163574, "mode": "test", "epoch": 94}
{"loss": 0.15383055806159973, "acc": 0.9444016814231873, "length": 3.7762746810913086, "mode": "train", "epoch": 95}
{"loss": 0.30456072092056274, "acc": 0.8996855616569519, "length": 3.7637362480163574, "mode": "test", "epoch": 95}
{"loss": 0.1534898579120636, "acc": 0.9457181096076965, "length": 3.7862493991851807, "mode": "train", "epoch": 96}
{"loss": 0.3008792996406555, "acc": 0.8954740166664124, "length": 3.7747251987457275, "mode": "test", "epoch": 96}
{"loss": 0.15187621116638184, "acc": 0.9473235011100769, "length": 3.7786595821380615, "mode": "train", "epoch": 97}
{"loss": 0.30424797534942627, "acc": 0.8957205414772034, "length": 3.7857143878936768, "mode": "test", "epoch": 97}
{"loss": 0.14799202978610992, "acc": 0.9459644556045532, "length": 3.8083384037017822, "mode": "train", "epoch": 98}
{"loss": 0.3328695297241211, "acc": 0.8834325671195984, "length": 3.846153736114502, "mode": "test", "epoch": 98}
{"loss": 0.14988823235034943, "acc": 0.9440715909004211, "length": 3.814596176147461, "mode": "train", "epoch": 99}
{"loss": 0.304429829120636, "acc": 0.8934627771377563, "length": 3.8296704292297363, "mode": "test", "epoch": 99}
{"loss": 0.1451779007911682, "acc": 0.9452396631240845, "length": 3.811964988708496, "mode": "train", "epoch": 100}
{"loss": 0.2979351282119751, "acc": 0.8977809548377991, "length": 3.769230842590332, "mode": "test", "epoch": 100}
test accuracy: 0.8157964944839478
